trajectorytools/trajectories/concatenate.py

```diff
@@ -58,20 +61,60 @@ def _concatenate_idtrackerai_dicts(traj_dicts):
     """
     traj_dict_cat = traj_dicts[0].copy()
     traj_cat = _concatenate_np(
-        [traj_dict["trajectories"] for traj_dict in traj_dicts]
+        [traj_dict["trajectories"] for traj_dict in traj_dicts], **kwargs
     )
     traj_dict_cat["trajectories"] = traj_cat
     return traj_dict_cat
 
 
-def from_several_idtracker_files(trajectories_paths, **kwargs):
+def pick_trajectory_file(session_folder):
+    """Select the best trajectories file
+    available in an idtrackerai session
+    """
+    trajectories = os.path.join(session_folder, "trajectories", "trajectories.npy")
+    trajectories_wo_gaps = os.path.join(session_folder, "trajectories_wo_gaps", "trajectories_wo_gaps.npy")
+
+    if os.path.exists(trajectories_wo_gaps):
+        return trajectories_wo_gaps
+    elif os.path.exists(trajectories):
+        return trajectories
+    else:
+        warnings.warn(f"No trajectory found in {session_folder}")
```

Comment:
I think this should raise a exception in this point (otherwise it would fail outside due to the lack of returned value)

Reply:
Corrected here https://github.com/fjhheras/trajectorytools/pull/43/commits/49df26b83e06520f9993d17b201c4e885eaae78d